fastlane/Fastfile

```diff
@@ -172,7 +172,7 @@ platform :ios do
   lane :code_freeze do |options|
     ios_codefreeze_prechecks(options)
 
-    ios_bump_version_release(skip_glotpress: true, skip_deliver: true)
+    ios_bump_version_release(skip_deliver: true)
```

Comment:
This is required because of the [breaking change](https://github.com/wordpress-mobile/release-toolkit/releases/tag/7.0.0) in the Release Toolkit `7.0.0`.